src/sdk/config_fs.js

```diff
@@ -244,11 +407,23 @@ class ConfigFS {
      * @returns {Promise<Object>}
      */
     async get_account_by_access_key(access_key, options = {}) {
-        const account_path = this.get_account_path_by_access_key(access_key);
-        const account = await this.get_config_data(account_path, options);
+        const account_path = this.get_account_or_user_path_by_access_key(access_key);
+        const account = await this.get_identity_config_data(account_path, options);
         return account;
     }
 
+    /**
+     * _remove_config_file_suffix removes suffix from a list entry name
+     * @param {string} entry
+     * @param {string} suffix
+     * @returns {string}
+     */
+    _remove_config_file_suffix(entry, suffix) {
```

Comment:
```suggestion
    _remove_config_file_suffix(entry_name, suffix) {
```

You have a builtin function:
path.parse(entry_name).name
For information:
https://stackoverflow.com/questions/4250364/how-to-trim-a-file-extension-from-a-string-in-javascript